src/configuration-client/application/src/main/java/org/niis/xroad/confclient/ConfClientCLIMain.java

```diff
@@ -87,9 +87,9 @@ private static CommandLine getCommandLine(String[] args) throws Exception {
         CommandLineParser parser = new DefaultParser();
         Options options = new Options();
 
-        options.addOption(OPTION_VERIFY_PRIVATE_PARAMS_EXISTS, false,
+        options.addOption("p", OPTION_VERIFY_PRIVATE_PARAMS_EXISTS, false,
```

Comment:
I think the verify_internal_configuration.sh script should be updated, because it contains 
"... org.niis.xroad.confclient.ConfClientCLIMain -verifyPrivateParamsExists "$@"" and the "-verifyPrivateParamsExists" will not be matched with long name.